waiter/src/waiter/scheduler.clj

```diff
@@ -889,10 +905,10 @@
                                   (concat healthy-instances))
             known-instance-ids' (->> known-instances' (map :id) set)
             previously-known-instance? (comp known-instance-ids :id)
-            new-instance-ids (->> known-instances'
-                                  (remove previously-known-instance?)
-                                  (sort instance-comparator)
-                                  (mapv :id))
+            new-instances (->> known-instances'
```

Comment:
Let's just log all of the instances from this collection with `:launch` as the event-type.